### ISSUE TITLE
Unable to use Elasticsearch 5 as a service because of too low vm.max_map_count

### ISSUE THREAD

**Issue by doubliez** [wercker/wercker#259]
I have a build pipeline which uses Elasticsearch 5 as a service:

```yaml
build:
  services:
    - elasticsearch:5.0
```

It worked properly with previous versions of Elasticsearch (2.x) but in version 5 several bootstrap checks have been added that will prevent Elasticsearch from starting if they fail.

One of them is failing in the Wercker environment, as you can see from the logs:

```
[2016-11-09T13:06:29,882][INFO ][o.e.b.BootstrapCheck     ] [x6_di-Y] bound or publishing to a non-loopback or non-link-local address, enforcing bootstrap checks
ERROR: bootstrap checks failed
max virtual memory areas vm.max_map_count [65530] likely too low, increase to at least [262144]
```

Fixing that implies changing the value of `vm.max_map_count` via `sysctl -w vm.max_map_count=262144`.

Is there any way to achieve that in Wercker?

**Comment by moritzzimmer** [comment]
Same for me, any updates on this?

**Comment by moritzzimmer** [comment]
Might be interesting to read: https://github.com/elastic/elasticsearch/pull/20511

**Comment by wimvanleuven** [comment]
Hey guys,

was redirected here from the the other ES pull request (elastic/elasticsearch#20511). As said over there, you can bypass the bootstrap checks to run in test mode with a single-node non-production setup:  setting http.host but not network.host. That however implies a non-clustered setup. Would that suit your use case?

**Comment by moritzzimmer** [comment]
@wimvanleuven correct me if I'm wrong, but this won't help if I like to use a *TransportClient* to connect to such a node?!

**Comment by wimvanleuven** [comment]
You are probably right ... My immediate reaction would be to change ES clients because you don't want to test the clients, but your own code; on the other hand you'll interject that the setup will not be the same as your real-life setup, ... deadlock, right?

**Comment by moritzzimmer** [comment]
Exactly, using different ES clients in test and production code feels wrong.

**Comment by termie** [comment]
it's pretty unclear to us what the results of changing this setting on the hosts would affect, I've gone to some significant length trying to convince elasticsearch not to rely on this in testing conditions but got rebuked, if you can provide convincing arguments as to why changing this value won't impair performance for us we can revisit this
